### PR TITLE
NAS-133327 / 25.04 / Remove @accepts from certain failover endpoints

### PR DESCRIPTION
**Pull request by yocalebo**
These are private endpoints that don't need to be decorated with @accepts.

### CODE REVIEW

File: src/middlewared/middlewared/plugins/failover_/zpool_cachefile.py [hunk 1]
```diff
@@ -18,8 +17,7 @@ class Config:
         private = True
         namespace = 'failover.zpool.cachefile'
 
-    @accepts(Str('event', enum=['MASTER', 'BACKUP', 'SYNC'], default='MASTER'))
-    def setup(self, event):
+    def setup(self, event: str = "MASTER"):
```
Comment (creatorcary) [inline src/middlewared/middlewared/plugins/failover_/zpool_cachefile.py:20]: Could we do `Literal["MASTER", "BACKUP", "SYNC"]`?

**Comment by creatorcary** [comment]
Are we not using `api_method(..., private=True)` for any of the remaining private endpoints to convert?

File: src/middlewared/middlewared/plugins/failover_/zpool_cachefile.py (hunk 1 above)
Comment (yocalebo) [reply src/middlewared/middlewared/plugins/failover_/zpool_cachefile.py]: Sure, done!

**Comment by yocalebo** [comment]
> Are we not using `api_method(..., private=True)` for any of the remaining private endpoints to convert?

Nah, not for these endpoints. The added complexity/overhead isn't needed for these.

**Review by creatorcary — APPROVED** [review]

**Pull request merged**